app/scripts/controllers/download.js

```diff
@@ -54,10 +55,17 @@ angular.module('lantern_www')
       return false;
     };
 
+    $scope.hideBetaLink = false; 
+
+    if ($translate.use() == constants.LANGS.fa_IR.code) {
+        $scope.hideBetaLink = true;
+    };
+
     $scope.trackEvent = function(type) {
         $window.ga('send', 'event', type, 
                    'click', osSniffer.os);
         if (type === 'download' && $translate.use() == constants.LANGS.fa_IR.code) {
+            $scope.irDownload = true;
```

Comment:
The ``irDownload`` variable and the div element have a two way binding. When you click on the ``Download`` link from the Persian language page, ``irDownload`` is set to true, and the ng-include is subsequently evaluated, fetching ``template/conversion.html``. Chrome network tab should confirm this.